snuba/datasets/table_storage.py

```diff
@@ -91,13 +93,8 @@ def get_replacement_topic_spec(self) -> Optional[KafkaTopicSpec]:
     def get_commit_log_topic_spec(self) -> Optional[KafkaTopicSpec]:
         return self.__commit_log_topic_spec
 
-    def get_all_topic_specs(self) -> Sequence[KafkaTopicSpec]:
-        ret = [self.__default_topic_spec]
-        if self.__replacement_topic_spec is not None:
-            ret.append(self.__replacement_topic_spec)
-        if self.__commit_log_topic_spec is not None:
-            ret.append(self.__commit_log_topic_spec)
-        return ret
+    def get_subscription_result_topic_spec(self) -> Optional[KafkaTopicSpec]:
+        return self.__subscription_result_topic_spec
```

Comment:
I think this is ok for the way we run subscription today. It basically ties an input stream to its commit log to the subscription result stream. We ma want to revisit this one day if we will allow multiple subscription results streams per input stream.